src/amo/components/AddonSummaryCard/index.js

```diff
@@ -49,7 +56,7 @@ export const AddonSummaryCardBase = ({
       </div>
       <div className="AddonSummaryCard-header-text">
         <h1 className="visually-hidden">{headerText}</h1>
-        <AddonTitle addon={addon} linkToAddon />
+        <AddonTitle addon={addon} linkToAddon linkSource={location.query.src} />
```

Comment:
similar to the above question, it's not going to resolve to `linkSource=null` is it?  That seems messy.

Reply:
That will be `undefined`

Reply:
This will be rendered to html, right?  If so wouldn't it be better resolving to `""` (or `linkSource` not being a dom node property if it's not set).

Reply:
It won't be rendered to HTML, it is a property that gets passed to another component but not all components render HTML with all the props that they receive.

In this case, `linkSource` will be used to add a `src` query param to the link rendered in the `AddonTitle` component, but only if it is not `undefined`.